charts/ssi-asr/templates/configmap-seeding-initialdata.yaml

```diff
@@ -0,0 +1,83 @@
+{{- /*
+* Copyright (c) 2024 Contributors to the Eclipse Foundation
+*
+* See the NOTICE file(s) distributed with this work for additional
+* information regarding copyright ownership.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License, Version 2.0 which is available at
+* https://www.apache.org/licenses/LICENSE-2.0.
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations
+* under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/}}
+
+{{- if .Values.migrations.seeding.useInitial -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  {{ .Release.Name }}-initial-data
+  namespace: {{ .Release.Namespace }}
+data:
+  authorities.json: |-
+    [
+        {
+            "bpn": "{{ .Values.operator.bpn }}"
+        },
+        {
+            "bpn": "BPNL00000003CRHL"
```

Comment:
We should discuss after my vacation if we should make this one also configurable. Maybe instead of calling operator.bpn, it's better to go with authorityOne, authorityTwo, but not sure

Reply:
adjusted :)